src/sass/base/_b-breakpoints.scss

```diff
@@ -5,12 +5,65 @@
 // after USWDS variables and before other partials.
 //===========================
 // @media single-keyword helpers
+
+// This is an override to fix a variable-naming collision between USWDS and Foundation.
+// Foundation declares these variables for legacy purposes, while USWDS uses them throughout
+// the whole codebase. This is a problem because Foundation's and USWDS's corresponding variables are
+// not the same units/types (Foundation's is defined in "em"'s), so USWDS is not able to compile.
+// Based on the order of our imports, USWDS should override Foundation's declarations and everything should be fine.
+// However, USWDS defines its variables using the "!default" flag, so Foundation's take precedence anyway.
+// These next lines redeclare USWDS's variables without that flag, so that the override works as we would expect.
 $small: new-breakpoint(min-width $small-screen $grid-columns-small);
 $medium: new-breakpoint(min-width $medium-screen $grid-columns-medium);
 $large: new-breakpoint(min-width $large-screen $grid-columns-large);
 
+// We do a custom override here, setting our site's medium breakpoint to 768 instead of how USWDS defines it at 600.
 $xsmall-screen:       320px; // QVGA display
 $medium-large-screen: 768px;
 $medium-screen:       $medium-large-screen;
 
 $medium: new-breakpoint(min-width $medium-large-screen 6);
+
+// This is an override on Neat's media mixin to create media queries for both screen and print
+// https://github.com/thoughtbot/neat/blob/v1.8.0/app/assets/stylesheets/grid/_media.scss
+@mixin media($query: $feature $value $columns, $total-columns: $grid-columns) {
```

Comment:
It's probably worth checking any uses of `@media()` in our code to see if affects a page that is meant to be printed (or already has print styles). This also potentially affects any uswds component that uses the grid and has print styling (I'm not sure if any do).

Reply:
It's actually even more confusing than how it looks here, because in the USWDS's imports, they first use this function as shown here for the grid SASS, then they import their utility SASS, which overrides `@media()` as shown here with a simpler `@media()` function that is used throughout the rest of the SASS. It was such a headache.

Reply:
Here's the new [media mixin](https://github.com/18F/web-design-standards/blob/develop/src/stylesheets/core/_utilities.scss#L46). So really, these changes should only affect the grid, because only the grid uses that version of the `@media` mixin. I didn't override the second `@media` mixin, which only applied to the screen.

This is way too confusing lol.

Reply:
I would guess that most pages are using Foundation's grid instead so I don't see anything too crazy happening to the print style, but yeah, you're definitely right.